boards/chronos/drivers/include/display_putchar.h

```diff
@@ -6,8 +6,8 @@
  * directory for more details.
  */
 
-#ifndef __DISPLAY_PUTCHAR_H
-#define __DISPLAY_PUTCHAR_H
+#ifndef DISPLAY_PUTCHAR_H
+#define DISPLAY_PUTCHAR_H
```

Comment:
for all other files until here you appended and underscore to the guard. Wouldn't it make sense to do it one way OR the other -> keep it consistent...